debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -58,25 +58,24 @@
 
     private static final String STATEMENTS_PLACEHOLDER = "#";
     private static final String DATABASE_NAME_PLACEHOLDER = "[#db]";
-    private static final String GET_MAX_LSN = "SELECT sys.fn_cdc_get_max_lsn()";
-    private static final String GET_LSN_TO_TIMESTAMP = "SELECT sys.fn_cdc_map_lsn_to_time(?)";
+    private static final String GET_MAX_LSN = "SELECT [#db].sys.fn_cdc_get_max_lsn()";
```

Comment:
It fails without db. I guess `sys` here is not just an alias to `master.sys` but more like a view.

Reply:
True. `fn_cdc_get_max_lsn()` definitely belongs to the current database, not globally to the instance. We were looking at something simpler like `sys.databases` when decided that the prefix before `sys` is not needed.